TTKDTrainingApp/src/redux/reducers/reactCamera.js

```diff
@@ -0,0 +1,21 @@
+import {ADD_RECORDED_VIDEO} from '../actionTypes';
+
+const initialState = {
+  recordedVideos: [],
```

Comment:
is this where the URIs would technically be ?

Reply:
This is a list of video objects that are the videos recorded by the user. The objects contain this structure:
```
{
"contentId", 
"deviceOrientation", 
"isRecordingInterrupted", 
"uri", 
"videoOrientation"
}
```
Everything here except for `contentId `is provided by `react-native-camera` when recording a video, and `contentId` is used to filter the videos when displaying them on a Content screen